### ISSUE TITLE
VSCode Typegen issue

### ISSUE THREAD

**Issue by faresite** [statelyai/xstate-tools#81]
Hello there 👋 

I'm new to using xstate and wanted to try it with the `tsTypes` typegen from the VSCode extension. However it doesn't seem to work for me, the xstate extension just crashes. I've tried on two different machines with the same result.

VSCode outputs the `The XState server crashed 5 times in the last 3 minutes. The server will not be restarted.` notification.

When I go to the terminal output I get the following errors:

```
internal/modules/cjs/loader.js:937
  throw err;
  ^

Error: Cannot find module 'prettier'
Require stack:
- /Users/XXXXXXX/.vscode/extensions/statelyai.stately-vscode-1.7.0/dist/server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:934:15)
    at Module._load (internal/modules/cjs/loader.js:779:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Module.require (internal/modules/cjs/loader.js:1006:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at ../../../packages/shared/dist/xstate-tools-shared.cjs.dev.js (/Users/XXXXXXX/.vscode/extensions/statelyai.stately-vscode-1.7.0/dist/server.js:88235:20)
    at __require (/Users/XXXXXXX/.vscode/extensions/statelyai.stately-vscode-1.7.0/dist/server.js:10:50)
    at ../../../packages/shared/dist/xstate-tools-shared.cjs.js (/Users/XXXXXXX/.vscode/extensions/statelyai.stately-vscode-1.7.0/dist/server.js:89006:25)
    at __require (/Users/XXXXXXX/.vscode/extensions/statelyai.stately-vscode-1.7.0/dist/server.js:10:50)
    at Object.<anonymous> (/Users/XXXXXXX/.vscode/extensions/statelyai.stately-vscode-1.7.0/dist/server.js:89015:36) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/XXXXXXX/.vscode/extensions/statelyai.stately-vscode-1.7.0/dist/server.js'
  ]
}
```

Prettier works fine in various projects, I've also tried to manually install prettier globally but that doesn't help. So not sure if it's anything to do with prettier.
Any ideas/help would be highly appreciated.

**Comment by mattpocock** [comment]
Looks like a bug in our latest release, will fix.

**Comment by faresite** [comment]
Hey 👋 
Quick update, I've decided to install a previous version of the extension (1.6.0) to check if there's an issue with the current one (1.7.0) as it looks quite fresh. And indeed 1.6 works just fine 👍 

No need to help me in this case as I'm happy to use whatever 1.6 provides at the moment as I'm exploring it.

**Comment by mattpocock** [comment]
Closing, should be fixed in 1.7.1
